packages/react-core/src/components/Slider/examples/Slider.md

```diff
@@ -69,25 +91,34 @@ class ContinuousInput extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      value: 50
+      value: 50,
+      valueCustom: 50
     };
 
-    this.onValueChange = value => {
+    this.onChange = value => {
         this.setState({
           value: value
         });
     };
+
+    this.onChangeCustom = value => {
+        this.setState({
+          valueCustom: value
+        });
+    };
   }
 
   render() {
     return (
       <>
         <Text component={TextVariants.h3}>Slider Value is: {this.state.value.toFixed(2)}</Text>
-        <Slider currentValue={this.state.value} onValueChange={this.onValueChange} />
+        <Slider value={this.state.value} onChange={this.onChange} />
         <br />
         <Slider
```

Comment:
This is the only example that doesn't display the selected value, not sure if that was intentional?